src/traversal/printRelayOSSQuery.js

```diff
@@ -246,7 +246,7 @@ function printDirectives(node) {
   return ' ' + directiveStrings.join(' ');
 }
 
-function printDirective({name,value}) {
+function printDirective({ name, value }) {
```

Comment:
Ok. Got it. But it's kinda of exception from proposed [airbnb codestyle](https://github.com/airbnb/javascript#5.1).